extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -16,12 +16,14 @@ import { Gmail } from '../../common/api/email-provider/gmail/gmail.js';
 import { Injector } from '../../common/inject.js';
 import { PubLookup } from '../../common/api/pub-lookup.js';
 import { Notifications } from '../../common/notifications.js';
+// note: only types are supported for OpenPGP.js or web-stream-tools, their function calls will fail
```

Comment:
This is because last time we tried to use OpenPGP.js in the content script, it didn't work (on Firefox). Actually, it would be nice to load it there. Then we wouldn't have to make so many calls to background_script (which we have to eventually deprecate anyway because of Google Chrome extension manifest v3. https://github.com/FlowCrypt/flowcrypt-browser/issues/1509)

I'll file an issue to try to attempt running openpgp.js in content script again, at which point this comment could be removed, and some things simiplified, in particular:
 - background calls from content script to background page that are only done because OpenPGP.js can be removed, and instead appropriate methods (which rely on OpenPGP.js) can be called directly
 - pgp_block.htm/ts that today calls into background page can instead call the content script when it needs OpenPGP.js, and those background call handlers can be moved from background page to content script

Reply:
https://github.com/FlowCrypt/flowcrypt-browser/issues/4906